src/index.js

```diff
@@ -23,8 +26,22 @@ class CreateDynamoDBGlobalTables {
     const region = provider.getRegion();
     const tableNames = this.getTableNames();
 
-    await Promise.all(tableNames.map(t => this.createGlobalTable(t, region)));
-    await Promise.all(tableNames.map(t => this.addReplica(t, region)));
+    if (region !== 'ca-central-1') {
```

Comment:
what about the current tables that we have in `ca-central-1`?

Reply:
The code in this `if` statement is only running things for version `2017.11.29` so we don't want it to run if the region is `ca-central-1` as it won't work.